app/pages/home-not-logged-in/research.jsx

```diff
@@ -13,7 +13,7 @@ counterpart.registerTranslations('en', {
     about: `
       Zooniverse gives people of all ages and backgrounds the chance to participate
       in real research with over 50 active online citizen science projects. Work with
-      1.6 million registered users around the world to contribute to research
+      2.7 million registered users around the world to contribute to research
```

Comment:
```suggestion
      millions of registered users around the world to contribute to research
```

Reply:
@shaunanoordin This is just a suggestion, but future-proofs the text from needing to make changes like this again in the future.

Consistent with @goplayoutside3's comment, I believe we removed "moving target" text like this from the copy for the new homepage design. And with the rollout of the new homepage coming soon, I had been ignoring issues like this. That said, easy to make this tiny fix, so go ahead!